packages/core/package.json

```diff
@@ -23,8 +23,8 @@
     "test:inspect": "node --inspect ../../node_modules/.bin/jest --watch --no-cache --runInBand",
     "dev:inspect": "node --inspect-brk -r ts-node/register src/scripts/dev.ts",
     "serve:inspect": "node --inspect-brk -r ts-node/register src/scripts/serve.ts",
-    "build": "../../node_modules/.bin/tsc --project ./tsconfig.build.json",
-    "build:watch": "../../node_modules/.bin/tsc --project ./tsconfig.build.json --watch",
+    "build": "tsc --project ./tsconfig.build.json",
+    "build:watch": "tsc --project ./tsconfig.build.json --watch",
```

Comment:
Just updated this line as well to use `tsc` directly.

Reply:
Uh... sorry I didn't explain this better. Using `tsc` only works if you call it from the root, which is what lerna does. But it doesn't work when you are in the folder (`/packages/core`). When you are there, you need to do `../../node_modules/tsc`. But that doesn't work on Windows. So I changed only the one that is used by lerna.

For that reason, I used `tsc` on the script that lerna uses and the other one on the one that you use when developing `core`.

It's a mess, I know. I was trying to avoid installing `typescript` on all packages because it's already on the root, but maybe that's the best solution.

Reply:
We have to either leave `build:watch` pointing to the root `tsc` or install `typescript` in the core.

Reply:
oh, I see. I ll just revert that commit in that case.